net-wireless/qflipper/files/qflipper-1.1.1_display_version.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/qflipper_common.pri b/qflipper_common.pri
+index e24d6acd..2f5dd504 100644
+--- a/qflipper_common.pri
++++ b/qflipper_common.pri
+@@ -25,20 +25,11 @@ unix:!macx {
+     error("Unsupported OS or compiler")
+ }
+ 
+-GIT_VERSION = $$system("git describe --tags --abbrev=0","lines", HAS_VERSION)
+-!equals(HAS_VERSION, 0) {
+-    GIT_VERSION = unknown
+-}
++GIT_VERSION = 1.1.1
+ 
+-GIT_COMMIT = $$system("git rev-parse --short=8 HEAD","lines", HAS_COMMIT)
+-!equals(HAS_COMMIT, 0) {
+-    GIT_COMMIT = unknown
+-}
++GIT_COMMIT = 9e1b07c9
+ 
+-GIT_TIMESTAMP = $$system("git log -1 --pretty=format:%ct","lines", HAS_TIMESTAMP)
+-!equals(HAS_TIMESTAMP, 0) {
+-    GIT_TIMESTAMP = 0
+-}
++GIT_TIMESTAMP = 1658935746
```

Comment:
why can't we let it autodetect so we don't have to change this patch every time?